File: CS/DataHelper.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Text;

namespace WindowsApplication1
{
    public class DataHelper
    {
        public static DataTable CreateTable(int RowCount)
        {
            DataTable tbl = new DataTable();
            tbl.Columns.Add("Name", typeof(string));
            tbl.Columns.Add("ID", typeof(string));
            tbl.Columns.Add("Number", typeof(string));
            tbl.Columns.Add("Test", typeof(string));
            for (int i = 0; i < RowCount; i++)
                tbl.Rows.Add(new object[] { String.Format("Name{0}", i), i, 3 - i, "Test" });
            return tbl;
        }
    }
}

File: VB/DataHelper.vb
Imports System.Data

Namespace WindowsApplication1

    Public Class DataHelper

        Public Shared Function CreateTable(ByVal RowCount As Integer) As DataTable
            Dim tbl As DataTable = New DataTable()
            tbl.Columns.Add("Name", GetType(String))
            tbl.Columns.Add("ID", GetType(String))
            tbl.Columns.Add("Number", GetType(String))
            tbl.Columns.Add("Test", GetType(String))
            For i As Integer = 0 To RowCount - 1
                tbl.Rows.Add(New Object() {String.Format("Name{0}", i), i, 3 - i, "Test"})
            Next

            Return tbl
        End Function
    End Class
End Namespace
